Tests/TensorFlowTests/LazyTensorTFFunctionBuilderTests.swift

```diff
@@ -0,0 +1,227 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import TensorFlow
+import CTensorFlow
+
+final class LazyTensorTFFunctionBuilderTests : XCTestCase {
+    override class func setUp() {
+        super.setUp()
+        _RuntimeConfig.useLazyTensor = true
+    }
+
+    func testSingletonInputs() {
+        let a = materializedLazyTensor(Tensor<Float>(10.0))
+        let w = Raw.identity(a)
+        XCTAssertEqual(tfFunction(w, "testSingletonInputs")!.description,
```

Comment:
one argument per line. (also for all the `XCTAssertEqual`s below)

Reply:
Done.